addons/test_mail/tests/test_mail_thread_internals.py

```diff
@@ -426,9 +426,17 @@ def test_suggested_recipients_default_create_value(self):
             'mobile_number': data_from_record_mobile,
         })
         suggestions = record._message_get_suggested_recipients()[record.id]
-        self.assertEqual(
+        self.assertItemsEqual(
             suggestions,
-            [(False, email, None, 'Customer Email', {'mobile': '+33199001015', 'phone': False})]
+            [
+                {
+                    'lang': None,
+                    'reason': 'Customer Email',
+                    'name': 'newpartner@example.com',
+                    'email': 'newpartner@example.com',
```

Comment:
Functional change? The name+email was not repeated twice in the previous version. Was it done in JS?

Reply:
Yes. In `parseEmail` function.